app.js

```diff
@@ -1,72 +1,93 @@
 // import functions and grab DOM elements
 import { makeStatsString } from './utils.js';
+
 const headDropdown = document.getElementById('head-dropdown');
 const middleDropdown = document.getElementById('middle-dropdown');
 const bottomDropdown = document.getElementById('bottom-dropdown');
+
+
 const headEl = document.getElementById('head');
 const middleEl = document.getElementById('middle');
 const bottomEl = document.getElementById('bottom');
 const reportEl = document.getElementById('report');
+
+
 const catchphrasesEl = document.getElementById('catchphrases');
 const catchphraseInput = document.getElementById('catchphrase-input');
 const catchphraseButton = document.getElementById('catchphrase-button');
 
 // set state for how many times the user changes the head, middle, and bottom
+let headCount = 0;
+let middleCount = 0;
+let bottomCount = 0;
 // set state for all of the character's catchphrases
+let catchphrases = [];
 
-headDropdown.addEventListener('change', () => {
+headDropdown.addEventListener('change', (e) => {
     // get the value of the head dropdown
-
+    const value = e.target.value;
```

Comment:
An alternative to using `e` here is to grab the value directly from the drop-down input element, which is declared above on lines 4-6.